package.json

```
@@ -4,10 +4,14 @@
   "description": "Official Top.gg Node SDK",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "npm ci && tsc",
+    "build": "tsc",
+    "build:ci": "npm i --also=dev && tsc",
     "docs": "npm run build && npm run md",
     "md": "jsdoc2md ./dist/structs/Api.js > ./docs/Api.md && jsdoc2md ./dist/structs/Webhook.js > ./docs/Webhook.md",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "lint": "eslint 'src/*/**/*.ts'",
+    "lint:ci": "eslint --output-file eslint_report.json --format json '*/**/*.ts'",
```

```suggestion
    "lint": "eslint src/**/*.ts",
    "lint:ci": "eslint --output-file eslint_report.json --format json src/**/*.ts",
```
Just tested it on my end, seems that these changes fix the errors I'm getting.